docs/getting-started/pingfederate.md

```diff
@@ -0,0 +1,158 @@
+# Getting Started - PingFederate
+
+<div class="banner" onclick="window.open('https://registry.terraform.io/providers/pingidentity/pingfederate/latest','');">
+    <img class="assets" src="../../img/logos/tf-logo.svg" alt="Terraform logo" />
+    <span class="caption">
+        <a class="assetlinks" href="https://registry.terraform.io/providers/pingidentity/pingfederate/latest" target=”_blank”>Registry</a>
+    </span>
+</div>
+
+## EARLY ACCESS
+
+As of January 2024, the PingFederate Terraform provider is an early access release. **This status indicates that the provider is in active development and is subject to breaking changes. While available in the Terraform Registry, use of this provider is not recommended for production use at this time.**
+
+## Requirements
+
+* Terraform CLI 1.4+
+* A running PingFederate server accessible over HTTPS, or Docker CLI to start one
+* If using Docker to start a PingFederate server, a DevOps license is required - [Register for the DevOps program here](https://devops.pingidentity.com/how-to/devopsRegistration/)
+
+## Start a PingFederate Docker image to be configured
+
+!!! warning "Using an Existing PingFederate Server"
+    If you already have a running PingFederate server that you can reach over HTTPS, you can skip this step. The provider can be used with any PingFederate server.
+
+First, start a PingFederate server using Docker. Your DevOps credentials will be read from the `${HOME}/.pingidentity/config` file. The HTTPS port (default `9999`) must be exposed.
+
+```shell
+docker run --name pingfederate_terraform_provider_container \
+  -d -p 9031:9031 \
+  -d -p 9999:9999 \
+  --env-file "${HOME}/.pingidentity/config" \
+  -e SERVER_PROFILE_URL=https://github.com/pingidentity/pingidentity-server-profiles.git \
+  -e SERVER_PROFILE_BRANCH=terraform-provider-pingfederate-1125 \
```

Comment:
is this a profile branch that we intend to maintain? Since the configs on the getting started docs are simple, could we use no profile or the getting started profile? No profile may require the user to login once